boostrap.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Start and configure a noid server.
+# This script is only intended for use in a development environment.
+
+NOIDSDIR="$HOME/goapps/noids"
+NOIDSLOG="$NOIDSDIR/dev.log"
+
+mkdir -p "$NOIDSDIR"
+touch "$NOIDSLOG"
+cat /dev/null > "$NOIDSLOG"
```

Comment:
It is clearing out the log file, without deleting the file, before the process starts. I was on the fence about the need for this.

I wanted a way of displaying the logging for `noids` on stdout _after_ the noid pool was initialized. `noids` is spawned in a fork and routes the log to disk. The dev pool of noids gets created, and then the contents of the log file is fed back to stdout.

The log file is ephemeral. Before I added this step repeatedly launching and killing `noids` would cause messages from previous instances of the server to show up when `bootstrap.sh` is first executed. I didn’t like this.